apps/protocol-frontend/src/components/MintModal.tsx

```diff
@@ -85,11 +89,9 @@ const MintModal = ({ contributions }: MintModalProps) => {
           date_of_submission: original.date_of_submission.toString(),
           engagementDate: original.engagementDate.toString(),
         };
-        mintContribution(originalClean, ipfsContentUri, setMintProgress);
+        mintContribution({ ...originalClean, ipfsContentUri });
```

Comment:
This is a good opportunity to make an ipfs hook and call it in the mint contribution hook

Reply:
Totally, but in different PR to be used in both mint & bulk mint.